src/seedu/addressbook/commands/TotalCommand.java

```diff
@@ -0,0 +1,21 @@
+package seedu.addressbook.commands;
+
+/**
+ * display the total number of persons stored in the address book
+ */
+public class TotalCommand extends Command {
+
+    public static final String COMMAND_WORD = "total";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": display the total number of persons stored in the address book.\n"
+            + "Example: " + COMMAND_WORD;
+
+    public static final String MESSAGE_TOTAL_IN_ADDRESSBOOK = "There is/are %1$s person(s) in the address book";
```

Comment:
I liked your usage of %1$s in the `MESSAGE_TOTAL_IN_ADDRESSBOOK` String constant :)